src/components/Gesture.svelte

```diff
@@ -235,14 +274,32 @@
             <i class="w-full h-full m-0 mt-4 p-2 fas fa-check fa-2x text-secondary" />
           </div>
           <StandardButton
-            onClick={recordClicked}
+            onClick={() => {
+              showCountdown = true;
+              countdownStart()}}
```

Comment:
If you move the `showCountdown = true` into the `countdownStart` function, this can be simplified.
```suggestion
            onClick={countdownStart}
```